common/lib/xmodule/xmodule/js/spec/video/general_spec.js

```diff
@@ -27,7 +27,7 @@
                     });
 
                     it('set the elements', function () {
-                        expect(this.state.el).toEqual('#video_id');
+                        expect(this.state.el).toHaveId('video_id');
```

Comment:
@symbolist `expect(this.state.el).toBe($('#video_id')[0]);` would be more appropriate as it would match the whole element.